mb-main.go

```diff
@@ -89,8 +98,8 @@ func checkMakeBucketSyntax(ctx *cli.Context) {
 	if !ctx.Args().Present() {
 		cli.ShowCommandHelpAndExit(ctx, "mb", 1) // last argument is exit code
 	}
-	for _, arg := range ctx.Args() {
-		if strings.TrimSpace(arg) == "" {
+	if len(ctx.Args()) != 1 {
+		if strings.TrimSpace(ctx.Args().First()) == "" {
 			fatalIf(errInvalidArgument().Trace(), "Unable to validate empty argument.")
 		}
```

Comment:
This is only true if someone explicitly passed a space arg "    " with quotes. 

if len(ctx.Args()) != 1 {
if strings.TrimSpace(ctx.Args().First()) == "" {
 			fatalIf(errInvalidArgument().Trace(), "Unable to validate empty argument.")
 		}